api_app/models.py

```diff
@@ -181,3 +188,80 @@ def delete_file(sender, instance: Job, **kwargs):
     if instance.file:
         if instance.file:
             instance.file.delete()
+
+
+class CustomConfig(models.Model):
+    class PluginType(models.IntegerChoices):
+        ANALYZER = 1, "Analyzer"
+        CONNECTOR = 2, "Connector"
+
+    type = models.PositiveSmallIntegerField(choices=PluginType.choices)
+    attribute = models.CharField(max_length=128, blank=False)
+    value = models.CharField(max_length=128, blank=False)
+    organization = models.ForeignKey(
+        Organization, on_delete=models.CASCADE, blank=True, null=True
+    )
+    owner = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE, blank=False, null=False
+    )
+    plugin_name = models.CharField(max_length=128, blank=False)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        # unique_together
+        # index
+
+        constraints = [
+            models.UniqueConstraint(
```

Comment:
good thing this one.

Ah, remember that you'll need to merge the migrations in a single one at the end. Like we do for the commits in Git.